chainer/functions/array/tile.py

```diff
@@ -41,21 +44,23 @@ def backward(self, inputs, grads):
         elif ndim < len(reps):
             shape = (1,) * (len(reps) - ndim) + shape
 
-        if grads[0].shape == ():
+        gy, = grad_outputs
+        if gy.shape == ():
             # This case should be treated differently because numpy.num would
```

Comment:
This comment is no longer valid as `numpy.sum` (seems like a typo) is now removed.